src/main/java/tracko/logic/parser/order/SortOrderCommandParser.java

```diff
@@ -0,0 +1,34 @@
+package tracko.logic.parser.order;
+
+import static tracko.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import tracko.logic.commands.order.SortOrderCommand;
+import tracko.logic.parser.Parser;
+import tracko.logic.parser.exceptions.ParseException;
+import tracko.model.order.OrderDateTimeComparator;
+
+/**
+ * Parses input arguments and creates a new SortOrderCommand object
+ */
+public class SortOrderCommandParser implements Parser<SortOrderCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the SortOrderCommand
+     * and returns a SortCommand object for execution.
```

Comment:
I think SortCommand here should be SortOrderCommand

Reply:
Thank you! I have just fixed these issues in the latest commit!